go/consensus/tendermint/apps/registry/state/state.go

```diff
@@ -23,7 +25,7 @@ var (
 	// signedEntityKeyFmt is the key format used for signed entities.
 	//
 	// Value is CBOR-serialized signed entity.
-	signedEntityKeyFmt = keyformat.New(0x10, keyformat.H(&signature.PublicKey{}))
+	signedEntityKeyFmt = keyformat.New(0x10, keyformat.H(&staking.Address{}))
```

Comment:
Since addresses are hashed, additional hashing via `H(...)` shoud not be needed. Also below.

This should also simplify some of the stuff below (e.g., the need for `hashAddr`).